src/Command/ImportContributorsCommand.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Command;
+
+use App\Service\GitHub;
+use App\Service\PictureService;
+use Symfony\Component\Console\Attribute\Argument;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+#[AsCommand(
+    name: 'app:import-contributors',
+    description: 'Import contributors from a GitHub project and save them in the database.',
+)]
+class ImportContributorsCommand
+{
+    public function __construct(
+        private GitHub $github,
```

Comment:
Note: since this is a constructor-level dependency, even calling `app:import-contributors --help` causes an error to indicate that the token is required.
This may be avoided by making the `GitHub` service a dependency of the `__invoke` method, although I'm not sure if that also works for CLI commands (it works for controllers).

Reply:
Neither here - just wanted to point it out, but since there's no alternative anyways this works just fine.